packages/fluentui/docs/src/examples/components/SplitButton/Usage/SplitButtonAlignMenuRightExample.shorthand.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { SplitButton } from '@fluentui/react-northstar';
+import { FilesGifIcon, FilesIllustratorIcon, FilesPdfIcon, FilesPhotoshopIcon } from '@fluentui/react-icons-northstar';
+
+const items = [
+  {
+    key: 'pdf',
+    content: 'Export to PDF',
+    icon: <FilesPdfIcon />,
+  },
+  {
+    key: 'pds',
+    content: 'Export to PDS',
+    icon: <FilesPhotoshopIcon />,
+  },
+  {
+    key: 'gif',
+    content: 'Export as GIF',
+    icon: <FilesGifIcon />,
+    disabled: true,
+  },
+  {
+    key: 'eps',
+    content: 'Export to EPS',
+    icon: <FilesIllustratorIcon />,
+  },
+];
+
+const SplitButtonAlignMenuRightExampleShorthand = () => (
```

Comment:
```suggestion
const SplitButtonPositioningExampleShorthand = () => (
```

I suggest to rename this example

Reply:
Done